Wrappers/Python/cil/optimisation/functions/MixedL21Norm.py

```diff
@@ -83,48 +143,25 @@ def proximal(self, x, tau, out=None):
         where the convention 0 · (0/0) = 0 is used.
         
         """
-
-        # Note: we divide x/tau so the cases of both scalar and 
-        # datacontainers of tau to be able to run
-        
+                
+        tmp = x.pnorm(2)
+        if has_numba and isinstance(tau, Number):
+            resarray = _proximal_step_numba(np.asarray(tmp.as_array(), order='C', dtype=np.float32), 
+                                np.abs(tau, dtype=np.float32))
+            res = tmp
+            # may involve a copy if the data is not contiguous
```

Comment:
I think we can just manipulate the data here and not keep a copy. I don't see any harm in using:

`tmp = np.asarray(tmp.as_array(),...)`

Reply:
I think we need tmp to be a `DataContainer` not a numpy array.